app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -372,6 +372,30 @@ def test_targeting_with_locales_and_countries(self):
             ),
         )
 
+    def test_targeting_with_locales_and_countries_mobile(self):
+        locale_ca = LocaleFactory.create(code="en-CA")
+        locale_ro = LocaleFactory.create(code="ro")
+        locale_de = LocaleFactory.create(code="de")
+        locale_en = LocaleFactory.create(code="en")
+        targeting_config = NimbusExperiment.TargetingConfig
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=NimbusExperiment.Version.NO_VERSION,
+            firefox_max_version=NimbusExperiment.Version.NO_VERSION,
+            targeting_config_slug=targeting_config.TARGETING_MOBILE_NEW_USER,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+            locales=[locale_ca, locale_ro, locale_de, locale_en],
+        )
+        self.assertEqual(
+            experiment.targeting,
+            (
+                "is_already_enrolled || days_since_install < 7 "
```

Comment:
The point @jhugman made about wrapping this in parens will be handled separately by https://github.com/mozilla/experimenter/issues/7120